x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.tsx

```diff
@@ -43,26 +66,34 @@ export const Result: React.FC<Props> = ({ result, showScore }) => {
         defaultMessage: 'View document details',
       })}
     >
-      <article className="appSearchResult__content">
-        <ResultHeader resultMeta={resultMeta} showScore={!!showScore} />
-        <div className="appSearchResult__body">
-          {resultFields
-            .slice(0, isOpen ? resultFields.length : RESULT_CUTOFF)
-            .map(([field, value]: [string, FieldValue]) => (
-              <ResultField key={field} field={field} raw={value.raw} snippet={value.snippet} />
-            ))}
-        </div>
-        {numResults > RESULT_CUTOFF && !isOpen && (
-          <footer className="appSearchResult__hiddenFieldsIndicator">
-            {i18n.translate('xpack.enterpriseSearch.appSearch.result.numberOfAdditionalFields', {
-              defaultMessage: '{numberOfAdditionalFields} more fields',
-              values: {
-                numberOfAdditionalFields: numResults - RESULT_CUTOFF,
-              },
-            })}
-          </footer>
-        )}
-      </article>
+      {conditionallyLinkedArticle(
+        <>
+          <ResultHeader resultMeta={resultMeta} showScore={!!showScore} />
+          <div>
```

Comment:
FWIW, I know `appSearchResult__body` wasn't being used as a CSS class - my thought when leaving it there was to have it serve as a landmark indicator of sorts (i.e. following the header/body/footer pattern). If we don't want it in at all, IMO let's at least change the div to:

```suggestion
          <section>
```

So it's a little clearer that the wrapper is intentional & semantic and someone (a.k.a. future me on a cleanup rampage) doesn't see an empty div and attempt to delete it / convert it into a fragment

Reply:
Makes sense, I can add it back.

Reply:
9ee2357658a6bbc1e112bf501106a08a4259d452

Reply:
9ee2357658a6bbc1e112bf501106a08a4259d452